packages/api/test/e2e/promise-tx.spec.js

```diff
@@ -1,7 +1,7 @@
 // Copyright 2017-2019 @polkadot/api authors & contributors
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
-
+import {ExtrinsicEra} from '@polkadot/types/type';
```

Comment:
(nit) Empty line after `//` and before import

Reply:
done